Roman-Numeral-Converter/converter.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <!-- GooglE fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Quicksand" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Roman Numeral Conversion</title>
+</head>
+<body>
+<div class="main">
+    <h1>ROMAN NUMERAL CONVERTER</h1>
+    <!-- arbic input -->
+    <input id="arabicInput" type="text" name="arabic"  placeholder="Enter an arabic number :">
+    <!-- roman input -->
+    <input id="romanInput" type="text" name="roma" value="" placeholder="Enter a roman number">
+</div>
+<div style="margin-top: 40px;text-align:center;font-size:20px;">
+    <div class="made-with">
+        <h4 class="color" style="color:white;font-family: cursive;">Made with ❤️ by <a href="https://rajprem.me" style="color:white;font-family: cursive;">Prem Raj</a></h4>
+    </div>
```

Comment:
Mentioning your Personal detail in the public project doesn't seem to be a good practice. @khushi-purwar please look into it

Reply:
Yes, remove this line

Reply:
> Mentioning your Personal detail in the public project doesn't seem to be a good practice. @khushi-purwar please look into it

Done the same